api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -393,9 +396,15 @@ private User initializeUserIfNeeded() {
     // profile, since it can be edited in our UI as well as the Google UI,  and we're fine with
     // that; the expectation is their profile in AofU will be managed in AofU, not in Google.
 
-    userService.createUser(request.getProfile().getGivenName(),
+    userService.createUser(
+        request.getProfile().getGivenName(),
```

Comment:
On speaking with Lou, we are doing a field change to first name and last name, rather than given and family.

Reply:
Updated code and screenshots to cover the new labeling. I did not rename the objects themselves, just the labeling.

Reply:
Okay, that works. Do you mind making a tech debt ticket or something to update the fields to match?

Reply:
Can do.